docs/notebooks/qtt_example.py

```diff
@@ -114,51 +114,65 @@
 
 virts.VP2.set(-60)
 
-cc1= virts.VP1()
-cc2=virts.VP2()
-r=80
-scanjob = scanjob_t({'sweepdata': dict({'param': virts.VP1, 'start': cc1-100, 'end': cc1 + 100, 'step': 4.}), 'minstrument': ['keithley1.amplitude'], 'wait_time': 0.})
-scanjob['stepdata'] = dict({'param': virts.VP2, 'start': cc2 - r, 'end': cc2 +r, 'step': 2.})
+cc1 = virts.VP1()
+cc2 = virts.VP2()
+r = 80
+scanjob = scanjob_t({'sweepdata': dict({'param': virts.VP1, 'start': cc1 - 100, 'end': cc1 +
+                                        100, 'step': 4.}), 'minstrument': ['keithley1.amplitude'], 'wait_time': 0.})
+scanjob['stepdata'] = dict({'param': virts.VP2, 'start': cc2 - r, 'end': cc2 + r, 'step': 2.})
 data = qtt.measurements.scans.scan2D(station, scanjob)
 gates.resetgates(gv, gv, verbose=0)
 
 
-print('virtual and physical gates: ' + ','.join( '%.2f' % x for x in [virts.VP1(),virts.VP2(),virts.VP3(), gates.P1(), gates.P2(), gates.P3() ]) )
+print('virtual and physical gates: ' + ','.join('%.2f' %
+                                                x for x in [virts.VP1(), virts.VP2(), virts.VP3(), gates.P1(), gates.P2(), gates.P3()]))
```

Comment:
linenr 128 is too long (>120)

Also linenr 4 and 160 too long

Reply:
@QFer Done